ui/round/src/round.nvui.ts

```diff
@@ -23,6 +25,14 @@ export function initModule(ctrl: RoundController): NvuiPlugin {
       default: 'actions-board',
       storage: storage.make('nvui.pageLayout'),
     }),
+    deviceType: makeSetting<DeviceType>({
```

Comment:
maybe rename `deviceType`/`desktop`/`touchscreen` to `layout`/`classic`/`constrained` since (i think) these options are only relevant to touch devices.